devops/README.md

```diff
@@ -1,17 +1,211 @@
+# Package Installation
+We use the [Microsoft repository](https://packages.microsoft.com/) to distribute our packages. Below are the instructions to install the `osconfig` package on various operating systems.
```

Comment:
>We [](http://example.com/codeflow?start=0&length=2)

Add a blank line between the section titles and bodies. For consistency with the other MD documents in our project.